src/calibrate/controllers/calibrate.py

```diff
@@ -0,0 +1,36 @@
+import app
+from routes import api
+from flask import Blueprint, request, jsonify
+import logging
+import app
+import json
+import uncertainties.unumpy as unp
+from jobs import regression
+import os
+from pymongo import MongoClient
+import uncertainties.unumpy as unp
+
+
+calibrate_bp = Blueprint('calibrate_bp', __name__)
+
+
+# MONGO_URI = os.getenv("MONGO_URI")
+# client = MongoClient(MONGO_URI)
+# db = client['airqo_netmanager_staging_airqo']
+
+
+@calibrate_bp.route(api.route['calibrate'], methods=['POST', 'GET'])
```

Comment:
It is also generally good to stick to one HTTP verb for each endpoint. As for this case, a GET can just be enough. Leaving it open to two or more is technically possible but kinda not a good practice.

Reply:
okay , noted